docs/tutorial/quick-start.md

```diff
@@ -0,0 +1,38 @@
+# # Quick start
+
+## What you'll do
+
+- Deploy the charm.
+- Provide GitHub runners on OpenStack mode with images.
+
+## Requirements
+
+- GitHub runner running in OpenStack mode.
```

Comment:
Since this is the first step of the tutorial, I think you can remove this requirement